scripts/common.py

```diff
@@ -421,6 +421,13 @@ def check_prs_for_current_release_are_released(repo):
 def convert_to_posixpath(file_path):
     """Occassionally the Windows style file path just does not work, so we
```

Comment:
Converts a Windows style filepath to posixpath format. If the operating system is not Windows, this function does nothing.

Reply:
Updated